influxdb3_write/src/last_cache/mod.rs

```diff
@@ -1278,35 +1412,33 @@ impl LastCacheStore {
     /// for the cache.
     fn to_record_batch(
```

Comment:
This function no longer builds a new arrow schema on _every_ call. As a result, the arrow schema will only ever need to be rebuilt when new fields are added for caches that accept new fields, _or_ never for caches that have an explicit set of value columns. Furthermore, for the explicit case, it produces the value columns by iterating directly on the cache `IndexMap`, instead of iterating over the schema. The non-explicit case still needs to iterate over the schema and do a lookup to get column ID. Therefore, I suspect the explicit case will be considerably more performant.